Brass-Demons/Welcome-to-Nashville
# Ms event listener

# Description

- Altered buildResultsHTML function to loop through returned results from meetup API and append them to the DOM

-Attached event listeners to Results save buttons

-Converted code in generateItinerary to a function 

- Added an object to contain all saved search results and added a way to pass their values to the html-creating function using string interpolation

- Made api event key private

Fixes # (issue)

-Moves app closer to completion and allows for each part to feed into each other more effectively

-Protects api key


# Testing Instructions

-Make sure meetup search button is populating results to the DOM

-Make sure save button is displaying text on the DOM

-Try plugging in the name of your saved results into the itineraryObj and make sure it populates to the DOM 


- Pull down the feature branch with `git fetch --all`.
- Checkout into the branch to review the feature on your local server.

# Checklist:
